src/RVRP_data.jl

```diff
@@ -38,9 +42,11 @@ mutable struct Pickup
     id::String # If its part of a shipment, it has is own id anyway
     index::Int # Not given in JSON. If its part of a shipment, the index of the shipment
     location::Location
-    shipment_id::String # NULL if none
-    price_reward::Float64
-    capacity_request::Float64
+    shipment_id::String # NULL if pure Pickup request
+    product_id::String # optional : required only to implement conflicts
+    conflicting_product_ids::Vector{String}  # optional : required only to implement conflicts
+    price_reward::Float64  # optional : required only to implement the price collection variant
+    quantity::Float64 # the available commodity quantity, or the requested quantity to pickup
```

Comment:
why not having only "the requested quantity to pickup" ? it covers all cases I think